### DISCUSSION TITLE
Nerd fonts icon not showing up

### DISCUSSION THREAD

**Discussion by afdallah** [jesseduffield/lazygit#2031] [Q&A] [answered]
I believe I have used the correct nerd font as seen in this vim ui.
But somehow on lazygit the icon didn't show up.

I have this in my lazygit settings
```
gui:
  showIcons: true
```
Vim 
![image](https://user-images.githubusercontent.com/11598084/178260620-768eab3f-25d1-45fa-8f77-4d5c3372de5a.png)

Lazygit
![image](https://user-images.githubusercontent.com/11598084/178260833-12105196-cfa3-4286-b52c-de2fd80717db.png)

**Comment by mark2185** [comment, marked as answer]
What's `lazygit`'s version and could you try building current `master`?

**Reply by afdallah** [reply]
![image](https://user-images.githubusercontent.com/11598084/178278212-3fc56d4d-0040-4050-84fb-8b5e0fe26264.png)
Current version is 0.34

**Reply by mark2185** [reply]
I believe `v0.34` did not have icon support. Please try out current `master`.

**Reply by afdallah** [reply]
Cool!, I see it's working, thanks!
![image](https://user-images.githubusercontent.com/11598084/178338674-d359eb2c-12d3-47a4-844a-ac65dbf49edc.png)

Sorry, right now I can only run it by coming to the lazygit dir and run `./lazygit`
I'm trying to copy the file to /usr/bin folder but was not able to do it even in root mode.
How to build and install it properly?

**Reply by afdallah** [reply]
Oops, now it's working. I copy it to `/usr/local/bin`.
Thanks so much!

**Reply by mark2185** [reply]
You can always use `go install .` and add the destination folder to your `$PATH`, which is `~/go/bin/` by default, I think? After running it try searching for `lazygit` in `~/go` and add the destination folder to your `$PATH`.

Or just add the path to the cloned repository to your `$PATH` until `v0.35` is released.
